src/foraging_gui/Foraging.py

```diff
@@ -1070,7 +1071,8 @@ def keyPressEvent(self, event=None):
                         new = float(child.text())
                         if new != old:
                             logging.info('Changing parameter: {}, {} -> {}'.format(child.objectName(), old,new))
-
+                    '''
+                    
```

Comment:
Your update removes the logging of the changed parameters. Instead of commented out the code, can we fix the problem?

Reply:
Yes, we need to fix it. I haven't explored which field caused this crash, so I just commented them temporarily for using in the ephys rig.

Reply:
Can you post the output of the GUI log?